crates/ruff/Cargo.toml

```diff
@@ -70,6 +70,7 @@ thiserror = { version = "1.0.38" }
 toml = { workspace = true }
 typed-arena = { version = "2.0.2" }
 unicode-width = { version = "0.1.10" }
+unicode_names2 = { version = "0.6.0", git = "https://github.com/youknowone/unicode_names2.git", rev = "4ce16aa85cbcdd9cc830410f1a72ef9a235f2fde" }
```

Comment:
We could use clippy's [cargo lint](https://rust-lang.github.io/rust-clippy/master/index.html#multiple_crate_versions) but it would require us to first solve all existing duplicate dependencies, which may not be possible. 

```
 cargo tree -d
```

Reply:
@MichaReiser - Are you cool with merging this as-is or would you want to get cargo lint in place?

Reply:
I'm okay with merging as is. I don't think that setting up `cargo lint` is trivial. 

Another (non trivial) idea would be to build our binary in CI in release mode (nightly?) and issue a warning if the binary size increased significantly.